test/unit/api/test_api.py

```diff
@@ -100,15 +101,15 @@ def test_get_file_info(self):
     def test_get_file_info_by_name(self):
         self._authorize_account()
         bucket = self.api.create_bucket('bucket1', 'allPrivate')
-        bucket.upload_bytes(
+        new_file = bucket.upload_bytes(
             b'hello world',
             'file',
         )
         result = self.api.get_file_info_by_name('bucket1', 'file')
 
         expected_result = {
-            'fileId': '9999',
-            'fileName': 'file',
+            'fileId': new_file.id_,
+            'fileName': new_file.file_name,
```

Comment:
I think that hardcoding values (where possible) in tests is better than using something dynamic, since that "something" may be wrong. I'd prefer previous version :)